media-tv/plex-media-server/plex-media-server-1.14.1.5488.ebuild

```diff
@@ -0,0 +1,97 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker user
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+_APPNAME="plexmediaserver"
+_USERNAME="plex"
+_SHORTNAME="${_USERNAME}"
+_COMMIT="cc260c476"
+_FULL_VERSION="${PV}-${_COMMIT}"
+
+URI="https://downloads.plex.tv/plex-media-server"
+SRC_URI="
+	amd64? ( ${URI}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_amd64.deb )
+	x86? ( ${URI}/${_FULL_VERSION}/${_APPNAME}_${_FULL_VERSION}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="avahi system-openssl"
+
+RDEPEND="avahi? ( net-dns/avahi )
+	system-openssl? ( dev-libs/openssl:0 )"
+
+QA_DESKTOP_FILE="usr/share/applications/plexmediamanager.desktop"
```

Comment:
since you already patch it, is it possible to patch it so there is no violation?